Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN addgroup --system --gid 107 django \
  RUN apt-get-install.sh libpq5
 
 # Install a modern Nginx and configure
-ENV NGINX_VERSION=1.14.2 \
+ENV NGINX_VERSION=1.18.0 \
```

Comment:
is 1.20.2 not suitable?

Reply:
When I looked 1.18.0 was the latest version on both stretch and buster